addons/website_slides/static/src/js/slides_course_quiz.js

```diff
@@ -390,11 +390,11 @@
             }
             if (completed) {
                 this._disableAnswers();
-                new SlideQuizFinishModal(this, {
-                    quiz: this.quiz,
-                    hasNext: this.slide.hasNext,
-                    userId: this.userId
-                }).open();
+                this.call("dialog", "add", SlideQuizFinishDialog, {
+                quiz: this.quiz,
+                hasNext: this.slide.hasNext,
```

Comment:
You can pass an onClickNext props to use directly:
```suggestion
                onClickNext: this._onClickNext,
```